services/actions/notifier.go

```diff
@@ -198,6 +204,22 @@ func (n *actionsNotifier) IssueChangeLabels(ctx context.Context, doer *user_mode
 		hookEvent = webhook_module.HookEventPullRequestLabel
 	}
 
+	// Track removedLabels for the webhook payload
+	var removedAPILabels []*api.Label
+	if len(removedLabels) > 0 {
+		if err := issue.LoadRepo(ctx); err != nil {
+			log.Error("LoadRepo: %v", err)
+			return
+		}
+
+		removedAPILabels = make([]*api.Label, 0, len(removedLabels))
+		for _, label := range removedLabels {
+			removedAPILabels = append(removedAPILabels, convert.ToLabel(label, issue.Repo, doer))
+		}
+
+		ctx = context.WithValue(ctx, removedLabelsKey, removedAPILabels)
```

Comment:
It's better to change the notify function signature directly to make it explicated.